Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: build clean run
+
+current_dir := $(shell pwd)
+current_user := $(shell id -u)
+
+build:
+	docker run --rm -v $(current_dir):/opt/accountapp/ -u $(current_user):$(current_user) -w /opt/accountapp --entrypoint ./gradlew java:8 --no-daemon --info war
```

Comment:
While I find this approach weird (who doesn't have JDK8 locally now :smile:), the java container is deprecated, please rely on the `openjdk` container instead.